pre_build.py

```diff
@@ -1,13 +1,21 @@
 import os
 import subprocess
+from shutil import which
+from sys import exit
 
 
 def build() -> None:
     if os.environ.get("SKIP_PRE_BUILD", "") == "true":
         print("Skipping front end build...")
         return
+    if which("yarn") is None:
+        print(
+            "Locust requires the yarn binary to be available in this shell to build the web front-end.\nSee: https://docs.locust.io/en/stable/developing-locust.html#making-changes-to-locust-s-web-ui"
+        )
+        exit(1)
     print("Building front end...")
-    subprocess.run(["make", "frontend_build"])
```

Comment:
What's the reason for changing this? The make would run those two yarn commands right?

Reply:
It removes the dependency on make. Helps with some issues experienced here https://github.com/locustio/locust/issues/2794